debian/rules

```diff
@@ -11,7 +11,11 @@ export LIBEXECDIR = "$(LIBDIR)/nx/bin"
 export INCLUDEDIR = "/usr/include/$(shell dpkg-architecture -qDEB_BUILD_MULTIARCH)"
 
 %:
-	CONFIGURE="./configure --prefix=/usr --libdir=$(LIBDIR) --includedir=$(INCLUDEDIR) --libexecdir=$(LIBEXECDIR)" dh $@ --with quilt
+	CONFIGURE="./configure --prefix=/usr \
+                               --disable-silent-rules \
```

Comment:
> Uh... careful here. I wonder how that worked for you in the first place. (missing tabs in debian/rules).

I changed that now as it makes things look nicer in editors that show tabs.

However, technically, it does not make a difference as we are wrapping lines here anyway.